helm/dashboards/dashboards/shared/private/kong-connection-distribution.json

```diff
@@ -1,119 +1,339 @@
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "allFormat": "glob",
+        "allValue": ".*",
+        "current": {
+          "selected": false,
+          "text": "All",
+          "value": "$__all"
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "PBFA97CFB590B2093"
+        },
+        "definition": "label_values(kong_datastore_reachable{cluster_id=~\"$cluster\"},namespace)",
+        "hide": 0,
+        "includeAll": true,
+        "label": "Namespace",
+        "multi": false,
+        "name": "namespace",
+        "options": [],
+        "query": {
+          "query": "label_values(kong_datastore_reachable{cluster_id=~\"$cluster\"},namespace)",
+          "refId": "PrometheusVariableQueryEditor-VariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "current": {
+          "selected": false,
+          "text": "All",
+          "value": "$__all"
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "PBFA97CFB590B2093"
+        },
+        "definition": "label_values(kong_datastore_reachable{cluster_id=~\"$cluster\", namespace=~\"$namespace\"},service)",
+        "hide": 0,
+        "includeAll": true,
+        "label": "Proxy",
+        "multi": false,
+        "name": "proxy",
+        "options": [],
+        "query": {
+          "query": "label_values(kong_datastore_reachable{cluster_id=~\"$cluster\", namespace=~\"$namespace\"},service)",
+          "refId": "PrometheusVariableQueryEditor-VariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "allFormat": "glob",
+        "allValue": ".*",
+        "current": {
+          "selected": false,
+          "text": "All",
+          "value": "$__all"
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "PBFA97CFB590B2093"
+        },
+        "definition": "label_values(kong_datastore_reachable{cluster_id=~\"$cluster\",namespace=~\"$namespace\",service=~\"$proxy\"},pod)",
+        "hide": 0,
+        "includeAll": true,
+        "label": "Pod",
+        "multi": false,
+        "name": "pod",
+        "options": [],
+        "query": {
+          "query": "label_values(kong_datastore_reachable{cluster_id=~\"$cluster\",namespace=~\"$namespace\",service=~\"$proxy\"},pod)",
+          "refId": "PrometheusVariableQueryEditor-VariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      }
+    ]
+  },
+  "time": {
+    "from": "now-6h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Kong Connection Distribution Copy",
+  "uid": "bb61eb43-e475-4169-ae0b-81b2e884c176",
```

Comment:
why a new uid? that would break links to the dashboard.

Reply:
Didn't know that but makes sense. Thanks! 
Fixed.